R/pkgs/report.generation/tests/testthat/test-county_loading.R

```diff
@@ -92,7 +92,7 @@ test_that("Simulation loading works", {
             scenario_labels = 'baseline',
             incl_geoids = included_geoids,
             inference=FALSE
-    ))+2
```

Comment:
Why did this change?

Reply:
I think this is a question for @juanderone ... I didn't edit this. maybe this is an accidental carryover from an older branch